agents/mongodb/internal/profiler/aggregator/aggregator.go

```diff
@@ -99,6 +99,9 @@ func (a *Aggregator) Add(ctx context.Context, doc proto.SystemProfile) error {
 	// we had some activity so reset timer
 	a.t.Reset(a.d)
 
+	doc.Ns = strings.ToValidUTF8(doc.Ns, "")
```

Comment:
please add a comment why we are doing it.

Reply:
I mean that we have issue with converting invalid symbols to valid utf-8 symbols like `\uFFFD` and that we have issue with sending this data.